notebook/static/notebook/js/notebook.js

```diff
@@ -2055,11 +2082,11 @@ define(function (require) {
      * @param {integer} end - index of the last cell to execute (exclusive)
      */
     Notebook.prototype.execute_cell_range = function (start, end) {
-        this.command_mode();
```

Comment:
The command/edit mode stuff tends to all be in place for very specific reasons - changing them will break or change the user experience in subtle ways - often related to focus. Can you clarify why you removed this?